example/db-migration-tf/variables.tf

```diff
@@ -7,31 +7,30 @@ variable "permissions_boundary_arn" {
   type = string
 }
 
-variable "prefix" {
-  type = string
-}
-
-variable "pg_host" {
-  type = string
-}
-
-variable "pg_user" {
+variable "rds_access_secret_id" {
   type = string
+  default = null
```

Comment:
What happens if this is null?

Reply:
Nothing, generally.   These are overrides for a situation where we know the data persistence layer isn't going to have that output (or the right output) for reason - in *theory* if the lookup has no key and this isn't set, it passes null to the submodule.    For `rds_access_secret_id` TF should error as `rds_access_secret_id` is required.   

This *might* be YAGNI in retrospect.